recipes/linuxbridge.rb

```diff
@@ -54,3 +54,9 @@
 end
 
 include_recipe 'openstack-network::ml2_linuxbridge'
+
+# When iptables-ng creates new rules, restart linuxbridge-agent so that it generates the iptables rules after iptables
+# is restarted.
+edit_resource(:service, 'neutron-plugin-linuxbridge-agent') do
+  subscribes :restart, 'ruby_block[create_rules]'
```

Comment:
Also, is there a way to explicitly specify the *ruby_block[create_rules]* _from_ the *iptables/manage* ? I feel like create_rules is a pretty generic name.